sdk/test-utils/recorder/src/utils/requestBodyTransform.ts

```diff
@@ -0,0 +1,57 @@
+export type RequestBodyTransformsType = {
+  stringTransforms?: Array<(body: string) => string>;
+  jsonTransforms?: Array<(body: { [x: string]: unknown }) => { [x: string]: unknown }>;
+};
+/**
+ * Under construction
```

Comment:
Let's state what this method does exactly. I find it confusing.

Reply:
Does this method makes the whole recording ignore request body parameters? (I'm assuming it doesn't, but the code is a bit too hard to read at the moment).

Reply:
I'll add comments with walkthrough

Reply:
I love the changes you made! Thank you!